### PR TITLE
Add info to the config file sample in the README

### PR DESCRIPTION
**Pull request by victor-onofrei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +57,20 @@ InputFileName=users_list.csv
...
+OutputFileName=output_Resolve-PendingMigrations.csv
+ItemCountThreshold=100
+
+[Get-GroupsReport]
+FirstCompanyIdentifier=compA
+FirstCompanyName=CompanyA
+SecondCompanyIdentifier=compB
+SecondCompanyName=CompanyB
+CompanyIdentifierAttribute=Department
+
```
Comment (revolter) [inline README.md:73]: Remove this line

File: README.md [hunk 2]
```diff
@@ -52,6 +57,20 @@ InputFileName=users_list.csv
...
+OutputDir=Resolve-PendingMigrations
+OutputFileName=output_Resolve-PendingMigrations.csv
+ItemCountThreshold=100
+
+[Get-GroupsReport]
+FirstCompanyIdentifier=compA
+FirstCompanyName=CompanyA
+SecondCompanyIdentifier=compB
+SecondCompanyName=CompanyB
+CompanyIdentifierAttribute=Department
```
Comment (revolter) [inline README.md:72]: Line #35 should be updated too. Do you plan to add sample config values for all the scripts?

**Review by revolter — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (victor-onofrei) [reply README.md:72]: Nope, not for all scripts. We can thus say "for several scripts".

Comment (revolter) [reply README.md:72]: Ok

File: README.md [hunk 3]
```diff
@@ -73,6 +91,12 @@ Name | Alias
 `-OutputFileName` | OFN
 `-ExchangeObjects` | EO
 
+_Note: In order to leverage this setup, you need to initialize any new script with
+`begin {
+    . "$PSScriptRoot\Initializer.ps1"
+    $params = Invoke-Expression "Initialize-DefaultParams $args"
+}`
```
Comment (revolter) [inline README.md:98]:
```suggestion
_Note: In order to leverage this setup, you need to initialize any new script with:_

```pwsh
begin {
    . "$PSScriptRoot\Initializer.ps1"
    $params = Invoke-Expression "Initialize-DefaultParams $args"
}
```
```

**Review by revolter — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -73,6 +91,14 @@ Name | Alias
 `-OutputFileName` | OFN
 `-ExchangeObjects` | EO
 
+_Note: In order to leverage this setup, you need to initialize any new script with:_
+
+```pwsh
+begin {
+    . "$PSScriptRoot\Initializer.ps1"
+    $params = Invoke-Expression "Initialize-DefaultParams $args"
+}
```
Comment (revolter) [inline README.md:100]: This code block is not closed.

**Review by revolter — CHANGES_REQUESTED** [review]

**Review by revolter — APPROVED** [review]

**Pull request merged**